docs/ota-client-guide/modules/ROOT/pages/evaluation-to-prod.adoc

```diff
@@ -8,19 +8,26 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 endif::[]
 
 
-The procedures to deploy OTA Connect in production are a little more complex than the basic xref:getstarted::index.adoc[Get Started]  procedures.
+The procedures to deploy OTA Connect in a realistic production scenario are a little more complex than the basic xref:getstarted::index.adoc[Get Started] procedures. But, you don't need to commit the resources for a full production roll-out right at the beginning of the process.
```

Comment:
I think the "But," as the start of a sentence is less than ideal. "However," might be better.

Reply:
Good call, done.